controllers/api.controllers.js

```diff
@@ -0,0 +1,10 @@
+const { openEndpoints } = require("../models/api.models");
+
+function getEndpoints(request, response) {
+  //   return openEndpoints().then((results) => {
+  //     console.log(results);
+  response.status(200).send({ endpoints: openEndpoints() });
+  //});
+}
+
```

Comment:
There's no need for a function call here, you can send the endpoints like this:
`response.status(200).send({ endpoints })`